app/views/api_docs/reference/v1.0-pre/_reference.md

```diff
@@ -781,6 +781,7 @@ Trainee details
         "ethnic_group": "mixed_ethnic_group",
         "ethnic_background": "Another Mixed background",
         "lead_partner_urn": "900020"
+        "lead_partner_ukprn": "12345"
```

Comment:
`lead_partner_ukprn` was actually already listed there and in a bunch of places. I only had to update it here because it wasn't possible to POST it in the request until now.